layers/stateless/sl_spirv.cpp

```diff
@@ -1488,4 +1490,69 @@ bool SpirvValidator::ValidateConservativeRasterization(const spirv::Module &modu
     return skip;
 }
 
+bool SpirvValidator::ValidateShaderTensor(const spirv::Module &module_state, const spirv::EntryPoint &entrypoint, const spirv::StatelessData &stateless_data, const Location &loc) const {
+    bool skip = false;
+
+    // check only shaders
+    if (entrypoint.is_data_graph)
```

Comment:
```suggestion
    if (entrypoint.is_data_graph) {
        return skip;
     }
```

nit, agree or not, want it to be the same everywhere